src/UI/FSMPMenu.cpp

```
@@ -0,0 +1,1578 @@
+	{
+		hdt::g_locale = code;
+		hdt::saveUserSettings();
+		hdt::loc::load();
+		g_localeChangedLive = true;
+	}
+
+	// The Language picker: a globe icon plus a combo listing "Auto" and every installed locale by native name.
+	void languageCombo()
+	{
+		FontAwesome::PushSolid();
+		ImGuiMCP::TextColored(kAccent, "%s", fa::Language);
```

Clang (14.0.6)

[warning] do not call c-style vararg functions

(cppcoreguidelines-pro-type-vararg)